chapters/es/slides/chapter2_03_word-vectors-similarity.md

```diff
@@ -167,26 +167,23 @@ que contienen menos palabras irrelevantes.
 
 ```python
 # Carga uno de los modelos más grandes que contiene vectores
-nlp = spacy.load("en_core_web_md")
+nlp = spacy.load("es_core_news_md")
 
-doc = nlp("I have a banana")
+doc = nlp("Tengo una manzana")
 # Accede al vector a través del atributo token.vector
-print(doc[3].vector)
+print(doc[2].vector)
 ```
 
 ```out
- [2.02280000e-01,  -7.66180009e-02,   3.70319992e-01,
-  3.28450017e-02,  -4.19569999e-01,   7.20689967e-02,
- -3.74760002e-01,   5.74599989e-02,  -1.24009997e-02,
-  5.29489994e-01,  -5.23800015e-01,  -1.97710007e-01,
- -3.41470003e-01,   5.33169985e-01,  -2.53309999e-02,
-  1.73800007e-01,   1.67720005e-01,   8.39839995e-01,
-  5.51070012e-02,   1.05470002e-01,   3.78719985e-01,
-  2.42750004e-01,   1.47449998e-02,   5.59509993e-01,
-  1.25210002e-01,  -6.75960004e-01,   3.58420014e-01,
- -4.00279984e-02,   9.59490016e-02,  -5.06900012e-01,
- -8.53179991e-02,   1.79800004e-01,   3.38669986e-01,
-  ...
+
+[-0.162944  0.042666  0.405069 -0.884944  0.13951   1.37826  -0.807906
+ -0.432592 -0.747897  0.953742  0.90389  -0.514217  0.360039 -0.409261
+  1.11574  -0.407411  0.118361 -0.426352 -0.315689  0.027726  0.79418
+ -0.99135   0.147428  0.36956   0.547555 -0.023946 -2.024585 -0.122916
+  0.406145  0.911639  0.76197  -0.01012  -0.763252 -0.674616  0.04863
+  1.395744 -0.349407 -0.515107 -0.737187  0.870402 -0.16684  -0.587724
+  0.406655 -0.444897  0.788494 -0.096343 -0.452753 -0.892051 -0.818379
+ -2.062615]
```

Comment:
We should reformat this to be more similar to the English and have a `...` at the end, to not imply that this is the entire 300-dimensional vector.